ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.create-run.tsx

```diff
@@ -371,6 +375,23 @@ const CreateRunPage = () => {
               )}
             />
 
+            <FormField
+              control={form.control}
+              name='path'
+              render={({ field }) => (
+                <FormItem>
+                  <FormLabel>Path</FormLabel>
+                  <FormControl>
+                    <Input {...field} />
+                  </FormControl>
+                  <FormDescription>
+                    An optional subpath of the repository to clone.
+                  </FormDescription>
```

Comment:
I think it would make sense to add some hint about the format here. As a user I'd be uncertain whether I need to use `dir`, `/dir` or `./dir`, or if all of them would work the same.

Reply:
Please see my answer to another comment below.